### ISSUE TITLE
Find out what an event is about

### ISSUE THREAD

**Issue by GuillaumeAV** [assemblee-virtuelle/archipelago#176]
**Tension**
When I enter an event, for example smarketplace demo, I would like to be able to enter the event's [Subject](http://virtual-assembly.org/ontologies/pair-2021-summer/index-en.html#Subject), i.e. the **project** Smarketplace, in addition to entering a topic. 

**Proposal**
In the PAIR ontology, there's a dedicated [Object Property: about](http://virtual-assembly.org/ontologies/pair-2021-summer/index-en.html#about)
We'd like to add it so that we can link events to subjects such as projects, orgas and so on. 
What's missing, however, is the inverse relationship that would make it possible to identify when you're on a [subject](http://virtual-assembly.org/ontologies/pair-2021-summer/index-en.html#about: the events or ideas they're about. The label could be something generic like: "About ..." / "A ce sujet"

A description of alternatives you've thought of.
Not really .

**Comment by srosset81** [comment]
Yes that would be useful for the future Astro website ! And we should add this to projects as well !

> FYI, for `pair:Document`, we use `pair:documents` / `paid:documentedBy`
